aicsimageio/readers/czi_reader.py

```diff
@@ -507,7 +541,7 @@ def _getmetadataxmltext(self, findpath, default=None):
             return default
         return ref.text
 
-    def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
+    def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float, float, float]:
```

Comment:
This should just `Tuple[float]` which is already a "Tuple of floats"

Reply:
Hmm my IDE complained about it being a type mismatch. I just did some digging and I found this 
```
# For tuples of fixed size, we specify the types of all the elements
x: Tuple[int, str, float] = (3, "yes", 7.5)

# For tuples of variable size, we use one type and ellipsis
x: Tuple[int, ...] = (1, 2, 3)
```
which suggests that the correct type would be 
`Tuple[float, float, float]`
or 
`Tuple[float, ...]`

Thoughts?

Reply:
Oh. Interesting. Didn't know that was the behavior. Then great leave as is!